src/nsolid.cc

```diff
@@ -521,6 +521,38 @@ int CpuProfiler::get_cpu_profile_(SharedEnvInst envinst,
                                                    deleter);
 }
 
+int Snapshot::start_allocation_sampling_(SharedEnvInst envinst,
+                                         uint64_t sampleInterval,
+                                         int stackDepth,
+                                         v8::HeapProfiler::SamplingFlags flags,
```

Comment:
Use snake-case in c++
```suggestion
                                         uint64_t sample_interval,
                                         int stack_depth,
                                         v8::HeapProfiler::SamplingFlags flags,
```